PoroElastic/PoroElasticity.C

```diff
@@ -102,11 +102,24 @@ bool PoroElasticity::initElement (const std::vector<int>& MNPC,
   std::vector<int> MNPCu(MNPC.begin(),pStart);
   std::vector<int> MNPCp(pStart,MNPC.end());
 
+  if (elmInt.vec.size() < NSOL)
```

Comment:
Nei, denne if() må være der, fordi den nye PoroPracture arver PoroElasticity og denne kan resize vec til noe større enn NSOL (for å ta høyde for fasefeltet, bl.a.)